src/NUnitFramework/TestBuilder.cs

```diff
@@ -179,7 +179,7 @@ public static ITestResult ExecuteWorkItem(WorkItem work)
             // TODO: Replace with an event - but not while method is static
             while (work.State != WorkItemState.Complete)
             {
-#if PORTABLE || NETSTANDARD1_6
+#if PORTABLE
                 System.Threading.Tasks.Task.Delay(1);
 #else
                 Thread.Sleep(1);
```

Comment:
That impression matches my understanding which is that `Delay` means "schedule using a precise timer" and `Sleep` means "I have nothing particularly important to do, deprioritize me."